templates/vt_device_driver_template.c

```diff
@@ -0,0 +1,133 @@
+/* Copyright (c) Microsoft Corporation.
```

Comment:
In each of the functions where its required for someone to fill out the content, I wonder if it would be worth adding a "error"directive so that if they dont do it they get a compile error.